ontologies/Sierra_film.json

```diff
@@ -0,0 +1,180 @@
+{
+  "@context": "https://api.wellcomecollection.org/catalogue/v0/context.json",
+  "id": "8rm28aw",
+  "type": "Work",
+  "identifiers": [
+    {
+      "label": "b24993207",
+      "authority": "Sierra"
+    }
+  ],
+  "label": "Are health tests a good idea? An Horizon special",
+  "creators": [
+    {
+      "label": "Davies, Serena, television producer, television director",
+      "type": "Agent"
+    },
+    {
+      "label": "Mosley, Michael, television presenter",
+      "type": "Agent"
+    },
+    {
+      "label": "Levy, Daniel, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Jenkins, Paul, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Dymond, Duncan, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Dweck, Marc, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Wilson, Robin, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Heath, Iona, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Welch, H. Gilbert, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Ablin, Richard J., 1940-, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Gnanapragasam, Vincent, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Vance, Maggie, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Birney, Ewan, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Timmons, Jamie, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "Elwood, Peter, contributor",
+      "type": "Agent"
+    },
+    {
+      "label": "BBC Science, producer",
+      "type": "Agent"
+    }
+  ],
+  "isPartOf": [
+    {
+      "label": "Horizon",
+      "type": "Work"
+    }
+  ],
+  "publicationDate": {
+    "label": "2015",
+    "type": "Period"
+  },
+  "physicalDescription": "1 DVD (60 min.). sound, colour, PAL. 12 cm.",
+  "workType": "Videorecordings",
+  "format": [
+    {
+      "label": "Digital: optical: stereo"
+    },
+    {
+      "label": "PAL"
+    },
+    {
+      "label": "video file: DVD video: region 2"
+    },
+    {
+      "label": "two-dimensional moving image: content"
+    },
+    {
+      "label": "video: media"
+    },
+    {
+      "label": "videodisc: carrier"
+    }
```

Comment:
Why are these single item dictionaries and not just a flat list? When would `format` have an additional field?